src/components/ClaimMasterPanel.js

```diff
@@ -89,6 +94,10 @@ class ClaimMasterPanel extends FormPanel {
     return totalServices + totalItems;
   }
 
+  filterReferHF(options, currentHFUuid=this.props.userHealthFacilityFullPath.uuid){
+    return options.filter((option)=>option.uuid!==currentHFUuid)
```

Comment:
This function is not used at all, I see that you're passing an anonymous function to _filterOptions_ property.